sample/browser/package.json

```diff
@@ -10,14 +10,21 @@
     "demo": "webpack serve"
   },
   "devDependencies": {
-    "@wasm-tool/wasm-pack-plugin": "1.3.1",
-    "html-webpack-plugin": "4.3.0",
-    "webpack": "^4.0.0",
-    "webpack-cli": "^4.0.0",
-    "webpack-dev-server": "^3.11.2"
+    "@wasm-tool/wasm-pack-plugin": "1.6.0",
+    "html-webpack-plugin": "4.5.2",
+    "webpack": "4.46.0",
+    "webpack-cli": "4.7.0",
+    "webpack-dev-server": "4.9.2"
   },
   "dependencies": {
-    "@mattrglobal/jsonld-signatures-bbs": "0.10.0",
+    "@mattrglobal/jsonld-signatures-bbs": "link:../../",
     "jsonld-signatures": "5.0.1"
+  },
+  "resolutions": {
+    "webpack/**/glob-parent": "5.1.2",
+    "webpack-dev-server/sockjs-client/eventsource": "1.1.1",
+    "@mattrglobal/jsonld-signatures-bbs/**/node-forge": "1.3.0",
```

Comment:
This should be fixed in the main repo?

Reply:
Had a chat with Zed, keep the sample resolutions atm, likely the `resolutions` in main package not work in sample if we use link to the package.

Will update to use specific version after the main package published.